spec/lib/oauth/token_request_spec.rb

```diff
@@ -1,6 +1,12 @@
 require 'spec_helper_integration'
 
 module Doorkeeper::OAuth
+  describe ClientCredentialsRequest do
+    subject do
+      # TODO Do we need something here too?
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.